Interop/StubGenerator/src/main/kotlin/org/jetbrains/kotlin/native/interop/gen/StubIrElementBuilders.kt

```diff
@@ -57,6 +57,37 @@ internal class MacroConstantStubBuilder(
         }
         return listOf(declaration)
     }
+    }
+
+/*!
+Yet another mangler, particularly to avoid secondary clash, e.g. when a property in prototype (interface)
+is mangled and that will cause another clash in the class which implements this interface.
```

Comment:
How does this mangler exactly help to avoid such a clash?
(Simple test with property in the subclass shows me that it probably doesn't).